audit/projects/kubernetes-public/services/compute/project-info.json

```diff
@@ -154,6 +158,18 @@
     {
       "limit": 50,
       "metric": "EXTERNAL_VPN_GATEWAYS"
+    },
+    {
+      "limit": 1,
+      "metric": "PUBLIC_ADVERTISED_PREFIXES"
+    },
+    {
+      "limit": 10,
+      "metric": "PUBLIC_DELEGATED_PREFIXES"
+    },
+    {
+      "limit": 1024,
+      "metric": "STATIC_BYOIP_ADDRESSES"
```

Comment:
Are we used BYOIP from CNCF? Do the CNCF or Kubernetes community have an AS number to use with services at other cloud providers?